### PR TITLE
store id

### PR DESCRIPTION
**Pull request by mstein-ck**
LAB-4526

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +83,24 @@ This returns a base64 encoded PNG *without the data header*.
...
+    <tr>
+        <td>xInvoice</td>
+        <td>String</td>
+        <td>Invoice number</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>xStoreId</td>
+        <td>String</td>
+        <td>Store/Clerk/Server ID. <br /> This field can be used to route transactions.<br /> Contact Support for more details</td>
```
Comment (ymwymw) [inline README.md:95]:
```suggestion
        <td>Store ID. <br /> To support Multi-Merchant use cases.<br /> Contact Support for more details.</td>
```

File: public/sample.html [hunk 2]
```diff
@@ -103,6 +89,37 @@ <h2>Payment</h2>
...
+                    <input type="text" name="xInvoice" value="12345" />
+
+                    <label for="xStoreId"><strong>Store ID</strong></label>
+                    <input type="text" maxlength="4" name="xStoreId" value="1" />
+
+                    <!-- <label for="xStreet"><strong>Street</strong></label>
+                    <input type="text" name="xStreet" value="123 Main St" /> -->
+                    <!-- <label for="xZip"><strong>ZIP</strong></label>
+                    <br />
+                    <input type="num" name="xZip" value="11111" /> -->
```
Comment (ymwymw) [inline public/sample.html:110]: Commented on purpose?

Comment (mstein-ck) [reply public/sample.html:110]: Yes, I couldn't see it showing up on gateway rpt. Is the point of this only for semi-integrated? Or is this missing from my device config?

Comment (ymwymw) [reply public/sample.html:110]: can you try a keyed transaction with street/zip set?

Comment (ymwymw) [reply public/sample.html:110]: can you try a keyed transaction with street/zip set?

**Review by ymwymw — COMMENTED** [review]
Waiting for changes as discussed

File: public/sample.html (hunk 2 above)
Comment (mstein-ck) [reply public/sample.html:110]: Keyed on the pax? We don't support that.

**Comment by mstein-ck** [comment]
@ymwymw removing street and zip. We can discuss as a separate issue.

File: README.md [hunk 3]
```diff
@@ -89,6 +83,24 @@ This returns a base64 encoded PNG *without the data header*.
...
+    <tr>
+        <td>xInvoice</td>
+        <td>String</td>
+        <td>Invoice number</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>xStoreId</td>
+        <td>String</td>
+        <td>Store/Clerk/Server ID. <br /> This field can be used to support multi-merchant use cases.<br /> Contact <a href="https://www.cardknox.com/contact/integration-question/">Support</a> for more details</td>
```
Comment (ymwymw) [inline README.md:95]: DId we discuss that this should be Custom2?

**Review by ymwymw — COMMENTED** [review]
waiting on the xCustom02 changes

File: src/Pax/transactionApi.js [hunk 4]
```diff
@@ -37,9 +25,14 @@ export function getTransactionCommand(request) {
     ];
 
     if (['T00'].includes(command))
-        payload.push(getAVSInformation(request, transactionType) + FS + FS + FS + FS);
-    else
+        payload.push(getAVSInformation(request, transactionType));
```
Comment (ymwymw) [inline src/Pax/transactionApi.js:28]: Were the 4 `FS` values not needed?

Comment (mstein-ck) [reply src/Pax/transactionApi.js:28]: They're still there

Comment (ymwymw) [reply src/Pax/transactionApi.js:28]: Hard to follow but cool!!

**Review by ymwymw — APPROVED** [review]
